### ISSUE TITLE
Standard for Magnet Polarity

### ISSUE THREAD

**Issue by kurtabersold** [gridfinity-unofficial/gridfinity-unofficial.github.io#15]
Greetings,

I am just getting into gridfinity, and have noticed that there is no published standard on magnet polarity orientation. The thought of people building incompatible thingamabob organizers keeps me up at night. Any possibility to add in some recommendations for that?

**Comment by tosaddler** [comment]
I agree, and as I was about to start gluing in magnets my thought was to have North always facing up. Seems like the most logical solution.
